apps/io-func-wallet-solution-backend/src/infra/http/handlers/create-wallet-instance-attestation.ts

```diff
@@ -0,0 +1,51 @@
+import * as t from "io-ts";
+
+import { pipe } from "fp-ts/function";
+
+import * as H from "@pagopa/handler-kit";
+import * as RTE from "fp-ts/lib/ReaderTaskEither";
+import * as E from "fp-ts/lib/Either";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { sequenceS } from "fp-ts/lib/Apply";
+import { logErrorAndReturnResponse } from "../utils";
+import { createWalletInstanceAttestation } from "../../../wallet-instance-attestation";
+import { ValidationError } from "../../../validation";
+import { GRANT_TYPE_KEY_ATTESTATION } from "../../../wallet-provider";
+
+const WalletInstanceAttestationRequestPayload = t.type({
+  grant_type: NonEmptyString,
```

Comment:
**nitpick**: Validating against literal there's no need to check the grant type below

```suggestion
  grant_type: t.literal(GRANT_TYPE_KEY_ATTESTATION),
```